### ISSUE TITLE
samples

### ISSUE THREAD

**Issue by drakon660** [tye-v2/tye2#36]

**Comment by razvangoga** [comment]
@drakon660 should we move the samples in their own repo?

**Comment by drakon660** [comment]
move?

**Comment by razvangoga** [comment]
yeah - was wandering if we should move the contents of the **/samples** folder to a dedicated repo **tye2-samples**

**Comment by drakon660** [comment]
Why not. But after publication of 0.12, so we could have reference to nuget package instead to a project?

**Comment by razvangoga** [comment]
agree
